users/models.py

```diff
@@ -62,3 +62,32 @@ class Profile(TimeStampedModel):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     bio = models.TextField(max_length=500, blank=True)
     image = models.ImageField(upload_to="profile_pics", blank=True)
+
+
+class UserFollowing(TimeStampedModel):
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["follower", "followed"],
+                name="unique_following",
+            )
+        ]
+        ordering = ["-created_at"]
+
+    follower = models.ForeignKey(
+        "User",
+        related_name="followed",
```

Comment:
maybe you can rename followed to following like instagram, it makes more sense

Reply:
changed